app/proguard-rules.pro

```diff
@@ -45,6 +45,18 @@
     boolean ANDROID_DETECTED return true;
 }
 
+####################################################################################################
+# Remove logs from
+####################################################################################################
+-assumenosideeffects class android.util.Log {
+    public static boolean isLoggable(java.lang.String, int);
+    public static int v(...);
+    public static int i(...);
+    public static int w(...);
+    public static int d(...);
+    public static int e(...);
```

Comment:
agi mentions in the issue that we shouldn't remove error or warning logs.

Reply:
This is just a draft, put up before any answers to that thread.

Reply:
+1 on landing this. Looks like we also need to capture `android.util.Log.println` which is used by A-C's LogSink:
https://github.com/mozilla-mobile/fenix/blob/d767cd199ede1364d5f69dc7e7ddc7d4fbb9b24a/app/src/main/java/org/mozilla/fenix/FenixApplication.kt#L117

https://github.com/mozilla-mobile/android-components/blob/master/components/support/base/src/main/java/mozilla/components/support/base/log/sink/AndroidLogSink.kt#L31

Reply:
OK, based on the discussion, we do want to strip just debug logs as a first step. Just need to find way to exclude A-C's debug logs as well, which is trickier as A-C is using `println` directly.

We could hook up a custom LogSink in Fenix release builds and filter out debug logs there?